css/templates/e-commerce/page-vars.css

```diff
@@ -15,12 +15,27 @@
  */
 
 :root {
+  --h1: 3.5rem;
```

Comment:
We could potentially use the following classes:
```
<h3 class="ampstart-title-lg">Large Title</h3>
<h3 class="ampstart-title-md">Title Text</h3>
<h3 class="ampstart-title-sm">Small Title</h3>
<h3 class="ampstart-subtitle">Subtitle text</h3>
<label class="ampstart-label">Label Text</label>
```

However they wouldn't be semantic since the subtitle would potentially be used for the title on narrow. We would also need more than 3 titles.  

I like how h1-h7 allow us to be super flexible whilst keeping the markup semantic. It's very readable.